dist/react-selectable-extended.js

```diff
@@ -108,6 +108,8 @@ return /******/ (function(modules) { // webpackBootstrap
 
 	function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
 
+	function _objectWithoutProperties(obj, keys) { var target = {}; for (var i in obj) { if (keys.indexOf(i) >= 0) continue; if (!Object.prototype.hasOwnProperty.call(obj, i)) continue; target[i] = obj[i]; } return target; }
```

Comment:
yeah, TBH i'm not even sure why we commit the dist/ stuff - but i'm just following what the original plugin did :shrug: